SolutionInfo.cs

```diff
@@ -6,10 +6,8 @@
 [assembly: AssemblyVersionAttribute("0.19.0")]
 [assembly: AssemblyFileVersionAttribute("0.19.0")]
 [assembly: ComVisibleAttribute(false)]
-namespace System
-{
-    internal static class AssemblyVersionInformation
-    {
+namespace System {
+    internal static class AssemblyVersionInformation {
```

Comment:
I think this is a side-effect of `.\build FormatCode` - I'm tracking this in https://github.com/shiftkey/Octokit.CodeFormatter/issues/2